### PR TITLE
adding details in Sign up page and  validate when naming pet UI

### PR DESCRIPTION
**Pull request by Balloii**

### CODE REVIEW

File: lib/App.dart [hunk 1]
```diff
@@ -30,7 +30,7 @@ class App extends StatelessWidget {
       debugShowCheckedModeBanner: false,
       initialRoute: checkCurentUser(),
       routes: {
-        "/": (context) => const SignIn(),
+        "/": (context) => const SignUp(),
```
Comment (aofattaporn) [inline lib/App.dart]: in this line pls to convert back to `SignIn()` page

File: lib/src/blocs/pet/petBloc.dart [hunk 2]
```diff
@@ -19,17 +19,23 @@ class petBloc extends Bloc<petEvent, petState> {
...
-      Future<String> response = petRepository.NamingPetEvent(event._PetName);
-
-      //ยิงหลังบ้านสำเร็จ
-      // 200 -> return LoadedNamingPet
-      if (await response == "Naming pet successfully"){
-        emit(LoadedNamingPet());
-      }
-      // 201 -> return ErrorNamingPet
-      else{
+      print(event._PetName);
```
Comment (aofattaporn) [inline lib/src/blocs/pet/petBloc.dart]: if you have already checked this, and not use this line. pls remove this

**Review by aofattaporn — COMMENTED** [review]
in overview, you have a little bit code that not clean. pls check my review before push again

**Comment by aofattaporn** [comment]
pls don't forget to review this code ( feature/validate_namingPet + feature/add detail in SignIn )

@teamchz  @Balloii  @sura1542  @thunyathep

File: lib/src/screens/autheentication/SignUp.dart [hunk 3]
```diff
@@ -144,126 +145,242 @@ class SignUpState extends State<SignUp> {
...
+                  //                   fontWeight: FontWeight.bold,
+                  //                 ),
+                  //               ),
+                  //             ),
+                  //           ),
+                  //         ],
+                  //       ),
+                  //     ),
+                  //   ],
+                  // ),
```
Comment (thunyathep) [inline lib/src/screens/autheentication/SignUp.dart]: If you didn't use this pls delete it.

**Review by thunyathep — COMMENTED** [review]
same Aof

**Comment by teamchz** [comment]
Same as thun

**Review by teamchz — APPROVED** [review]

**Review by aofattaporn — APPROVED** [review]
LGTM

**Pull request merged**